sonar-plugin-api/src/main/java/org/sonar/api/measures/CoreMetrics.java

```diff
@@ -362,82 +362,109 @@
     .setDomain(DOMAIN_COMPLEXITY)
     .create();
 
+  /**
+   * @deprecated since 6.7
+   */
+  @Deprecated
   public static final String FILE_COMPLEXITY_KEY = "file_complexity";
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)